CMP1903M Assessment 1 Base Code/Input.cs

```diff
@@ -8,57 +8,72 @@ namespace CMP1903M_Assessment_1_Base_Code
 {
     public class Input
     {
+        private string? _input;
+        public string input { get { return (string)input; } }
```

Comment:
These lines may be simplified.
C# Standard recommends public properties use PascalCase.
```suggestion
        public string Input { get; }
```